R/knitr-methods.R

```diff
@@ -20,6 +20,9 @@ registerMethods <- function(methods) {
 }
 
 .onLoad <- function(...) {
+  # warn if the version of shiny is lower than what was specified in DESCRIPTION
+  checkShinyVersion(error = FALSE)
```

Comment:
On the other hand, the note probably won't get raised on the CRAN check machines, because they will have the new version of Shiny on there before htmlwidgets is submitted.